pages/[...slug].tsx

```diff
@@ -23,6 +23,9 @@ const Breadcrumbs = dynamic(() =>
   import('../src/components/navigation/Breadcrumbs')
 )
 const Topic = dynamic(() => import('../src/components/content/Topic'))
+const NewsletterPopup = dynamic(() =>
+  import('../src/components/NewsletterPopup')
```

Comment:
You might want to disable SSR for that component (see https://nextjs.org/docs/advanced-features/dynamic-import#with-no-ssr)

Reply:
nice!